documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -6,10 +6,41 @@
 [id='assembly-upgrade-{context}']
 = Upgrading Strimzi
 
-Strimzi can be upgraded from a lower version to a higher version to take advantage of new features and enhancements, performance improvements, and security options.
+Strimzi can be upgraded from a given version to a later version to take advantage of new features and enhancements, performance improvements, and security options.
 
-As part of the upgrade, you upgrade Kafka to a higher supported version. 
-Each Kafka release introduces new features, improvements, and bug fixes.
+As part of the upgrade, you upgrade Kafka to a later supported version. 
+Each Kafka release introduces new features, improvements, and bug fixes to your Strimzi deployment.
+
+Strimzi can be xref:assembly-downgrade-{context}[downgraded] to the previous version if you encounter issues with a newer version.
+
+.Upgrade paths
+
+Two upgrade paths are possible:
+
+Incremental:: 
+Upgrading Strimzi from the current version to the next version.
++
+For example, upgrading from Strimzi 0.20.1 to 0.21.0.
+
+Multi-version::
+Upgrading Strimzi from the current version to any later version within a single upgrade (skipping one or more versions).
++
+For example, upgrading from Strimzi 0.19.0 directly to Strimzi 0.21.0.
+
+.Kafka version support
+
+You can review supported Kafka versions in the link:https://strimzi.io/downloads/[Supported versions^] table.
+
+* The *Operators* column lists all released Strimzi versions (the Strimzi version is often called the "Operator version"). 
+
+* The *Kafka versions* column lists the Kafka versions that are supported by each Strimzi version. 
+
+//When upgrading to an Operator version that drops support for the Kafka version you are upgrading from, the upgraded Cluster Operator returns an error message, for example:
```

Comment:
TBC: Decide if this information is better here, or in the Kafka upgrade procedure.